src/test/java/com/puppycrawl/tools/checkstyle/internal/XDocsPagesTest.java

```diff
@@ -583,6 +584,7 @@ private static String getCheckPropertyExpectedTypeName(Class<?> clss, Object ins
         final String instanceName = instance.getClass().getSimpleName();
         String result = null;
 
+        // noinspection IfStatementWithTooManyBranches
```

Comment:
Intelij inspections should be suppressed in javadoc comments to not pollute the code.